CHANGELOG.md

```diff
@@ -7,6 +7,17 @@ Future Todo List
 ------------------------------
 - Make typography and utility classes silent extenders (so that they can be extended by components without importing all utility classes).
 
+v7.0.0
+------------------------------
+*January 25, 2022*
+
+### Changed
```

Comment:
Can you add the new mappings here too - so x2 now equals b (or whatever it actually is). Will just make the upgrade easier for when people need to replace those values in their repos

Reply:
Yup will do.

Reply:
Added a table showing old vs new.